src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -106,6 +110,13 @@ export default class GroupComparisonPage extends React.Component<
         this.urlWrapper.destroy();
     }
 
+    @computed get customTabs() {
+        return prepareCustomTabConfigurations(
+            getServerConfig().custom_tabs,
+            'COMPARISON_PAGE'
```

Comment:
So, you extended the custom tabs to Comparison Page? Nice would be good to make that clean in docs and release notes.